Content.Server/Kitchen/EntitySystems/KitchenSpikeSystem.cs

```diff
@@ -92,8 +92,9 @@ private void OnInteractUsing(EntityUid uid, KitchenSpikeComponent component, Int
             if (!Resolve(uid, ref component) || !Resolve(victimUid, ref butcherable))
                 return;
 
-            component.MeatPrototype = butcherable.SpawnedPrototype;
-            component.MeatParts = butcherable.Pieces;
+            // TODO VERY SUS
+            component.MeatPrototype = butcherable.SpawnedEntities[0].PrototypeId;
+            component.MeatParts = butcherable.SpawnedEntities[0].Amount;
```

Comment:
i could not think of a better way to do this without reworking kitchen spike entirely

Reply:
actually i did just have an idea